lib/routes/api/admin.ts

```diff
@@ -26,6 +26,7 @@ export default (crowi: Crowi, app: Express, form) => {
 
   router.get('/admin/notification', Admin.api.notification.index)
   router.post('/admin/notification/slackSetting', csrf, form.admin.slackSetting, Admin.api.notification.slackSetting)
+  router.post('/admin/notification/removeSlackSetting', csrf, Admin.api.notification.removeSlackSetting)
```

Comment:
Is it okay? I thought the naming convention of other routes is `resource.action`.
```suggestion
  router.post('/admin/notification/slackSetting.remove', csrf, Admin.api.notification.removeSlackSetting)
```
In other words, I think the above example follows other routes naming.

Reply:
OK, an if we do so, `/slackSetting` should also be `/slackSetting.add`, I'll fix it.

Reply:
I see.

Reply:
fixed.